### PR TITLE
Add Desktoppr 0.3

### PR DESCRIPTION
**Pull request by franklouwers**
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/desktoppr.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask "desktoppr" do
+  version "0.3"
+  sha256 "9a2ef033fd05b3b559af91f8401897ab6311f1a79d71581502c2fa0e9c52e664"
+
+  url "https://github.com/scriptingosx/desktoppr/releases/download/v#{version}/desktoppr-#{version}.pkg"
+  appcast "https://github.com/scriptingosx/desktoppr/releases.atom"
+  name "desktoppr"
+  desc "Simple command-line tool to set the desktop picture"
```
Comment (miccal) [inline Casks/desktoppr.rb:8]:
```suggestion
  desc "Command-line tool to set the desktop picture"
```

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @franklouwers.  

Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

**Comment by vitorgalvao** [comment]
From [faq/rejected_casks](https://github.com/homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> The app is both open-source and CLI-only (i.e. it only uses the `binary` artifact). In that case, and [in the spirit of deduplication](https://github.com/Homebrew/homebrew-cask/issues/15603), submit it first to [Homebrew/core](https://github.com/Homebrew/homebrew-core) as a formula that builds from source. If it is rejected, you may then try again as a cask (link us to the issue so we can see the discussion and reasoning for rejection).

In this case it uses a `pkg`, but it seems to only install a command-line tool, so the same logic applies.

**Comment by franklouwers** [comment]
> Thank you for your contribution to `homebrew-cask` @franklouwers.
> 
> Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

To the best of my knowledge, it doesn't have any preference or data-files, so a Zap wouldn't do anything. Does it need Nil-zap?

**Comment by franklouwers** [comment]
> From [faq/rejected_casks](https://github.com/homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):
> 
> > The app is both open-source and CLI-only (i.e. it only uses the `binary` artifact). In that case, and [in the spirit of deduplication](https://github.com/Homebrew/homebrew-cask/issues/15603), submit it first to [Homebrew/core](https://github.com/Homebrew/homebrew-core) as a formula that builds from source. If it is rejected, you may then try again as a cask (link us to the issue so we can see the discussion and reasoning for rejection).
> 
> In this case it uses a `pkg`, but it seems to only install a command-line tool, so the same logic applies.

Yes, it's a cli-tool, open-sourced. But as it's an XCode Project, it requires the full XCode to compile, not sure we support those in regular brews? If we do, could you please point me to a formula that I could use as inspiration?

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**